presto-main/src/main/java/com/facebook/presto/operator/WindowOperator.java

```diff
@@ -529,4 +530,11 @@ else if (!previousPairsWereEqual) {
         // the input is sorted, but the algorithm has still failed
         throw new IllegalArgumentException("failed to find a group ending");
     }
+
+    @Override
+    public void close()
+            throws Exception
+    {
+        driverWindowInfo = Optional.of(windowInfo.build());
```

Comment:
this will make window info available only after operator is finished. I think it would be better if we can have continuous window information similar as for hash collisions and others. This will enable as to display them continuously in GUI or `EXPLAIN ANALYZE`.